.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Small, unrelated change. This causes VSCode to build in a separate directory from `cargo build` so they don't compete for file locks.